naplib/naplab/process_ieeg.py

```diff
@@ -137,13 +137,19 @@ def process_ieeg(
         
```

Comment:
Change docstring for aud_channel_infer method to include description of the other options: 'crosscorr', 'envelope', 'interactive', and change default value